app/views/alchemy/essences/_essence_file_editor.html.erb

```diff
@@ -46,10 +46,8 @@
         class: 'edit_file',
         title: Alchemy.t(:edit_file_properties) %>
     </div>
-    <% if content.ingredient %>
-      <%= hidden_field_tag content.form_field_name(:attachment_id),
-        content.ingredient.id %>
-    <% end %>
+    <%= hidden_field_tag content.form_field_name(:attachment_id),
+      content.ingredient && content.ingredient.id %>
```

Comment:
Is the `attachment_id` param sent if `content.ingredient` is `nil`?
My question is: Doesn't Rails prevent the params from containing `nil` values coming from a form? Or is it for checkboxes only?